install.js

```diff
@@ -70,6 +70,24 @@ const installStaticFiles = (packageRoot, task) => {
   }
 };
 
+/**
+ * Replace a script inside `package.json` if it doesn't contain the expected default script.
+ * This ensures that users can add script snippets representing custom needs inside these scripts
+ * without having their changes overridden on installation. If the default script is not contained, however,
+ * it is inserted with everything else removed.
+ * This method mutates the `packageJson` argument passed in.
+ */
+const replaceScript = (packageJson, scriptName, scriptContents) => {
```

Comment:
How about a signature of:

```ts
const replaceScript = (scriptsMap: Map<string, string>) => packageJson => void
```

where `scriptsMap` is a `Map` of `"start", "webpack-dev-server"`, for example? That way this function gets invoked _once_ as opposed to [4 times](https://github.com/contiamo/operational-scripts/pull/24/files#diff-f16acefe4b6553580c43edab685f50f3R100) and is a _little_ more reusable.